terraform/application/outputs.tf

```diff
@@ -0,0 +1,17 @@
+# # Access the first hostname
+# output "claims_web_external_hostname" {
+#   value = module.web_application.hostname[0]
+# }
+
+# output "claims_web_external_hostname" {
+#   value = module.web_application.web_external_hostnames[0]
+# }
+
+# # Access the second hostname
+# output "placements_external_hostname" {
+#   value = module.web_application.hostname[1]
+# }
```

Comment:
can we remove the above comments if not using these?

Reply:
Fixed